CMakeLists.txt

```diff
@@ -149,7 +149,6 @@ auto_sources(hfiles "*.h" "RECURSE" "${FOLLY_DIR}")
 REMOVE_MATCHES_FROM_LISTS(files hfiles
   MATCHES
     "^${FOLLY_DIR}/build/"
-    "^${FOLLY_DIR}/experimental/exception_tracer/"
```

Comment:
Is this delta intended to be part of this pull request?

Reply:
Oh sorry that was unintentional. I got that from https://github.com/facebook/folly/pull/1484 which was needed to fix exception stack traces.